src/Libraries/RevitNodes/AnalysisDisplay/FaceAnalysisDisplay.cs

```diff
@@ -180,6 +181,7 @@ private void
         /// <param name="description">An optional analysis results description to show on the results legend.</param>
         /// <param name="unitType">An optional Unit type to provide conversions in the analysis results.</param>
         /// <returns>A FaceAnalysisDisplay object.</returns>
+        [NodeObsolete("SurfaceDataObsolete", typeof(Properties.Resources))]
```

Comment:
Hi @ZiyunShang does this attribute also hide the nodes?

Reply:
Yes, they have been hidden.